src/commands/deploy.ts

```diff
@@ -1,13 +1,10 @@
 import { Command, flags } from '@oclif/command';
 import chalk from 'chalk';
 import cli from 'cli-ux';
-import * as FormData from 'form-data';
-import * as fs from 'fs';
-import fetch from 'node-fetch';
-import { Response } from 'node-fetch';
-
-import { AppCompiler, AppPackager, FolderDetails } from '../misc';
-
+import * as Listr from 'listr';
+import { FolderDetails } from '../misc';
```

Comment:
Please add a blank line between npm imports and relative imports. Check the left deleted code for reference.